index.html

```diff
@@ -23,16 +22,30 @@
 </head>
 <body>
   <header class="d-flex d-flex-column">
+    <!-- Here Starts Mobile Menu -->
+    <nav class="mobile-menu display-none">
+      <div class="close">
+        <a href="#" id="toggle-x">X</a>
+      </div>
+      <div class="d-flex">
+        <ul id="small-nav">
+          <li><a href="#welcome-section">Portfolio</a></li>
+          <li><a href="#about-me">About</a></li>
+          <li><a href="#contact-me">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+    <!-- Here Starts Desktop Menu -->
     <nav class="d-flex">
       <ul id="big-nav" class="t-center">
-        <li>Portfolio</li>
-        <li>About</li>
-        <li>Contact</li>
+        <li><a href="#welcome-section">Portfolio</a></li>
+        <li><a href="#about-me">About</a></li>
+        <li><a href="#contact-me">Contact</a></li>
       </ul>
       <a href="#" id="welcome-link">
         <p id="welcome-text">Welcome</p>
       </a>
-        <img src="img/Union.png" alt="" id="toggle-menu">
+        <a class="humburger-btn"><img src="img/Union.png" alt="" id="toggle-menu"></a>
```

Comment:
- [ ] OPTIONAL
It's a good practice to provide `alt` names for your images so as to improve the accessibility features of your site.  Kindly endeavor to add `alt` names to **all** your images.